transforms/arrayToList.js

```diff
@@ -0,0 +1,32 @@
+/** @license ISC License (c) copyright 2017 original and current authors */
+/** @author Ian Hofmann-Hicks (evil) */
+
+const isFunction = require('../predicates/isFunction')
+const isArray = require('../predicates/isArray')
+
+const curry = require('../helpers/curry')
+
+const List = require('../crocks/List')
+
+// arrayToList : e -> [ a ] -> List a
```

Comment:
Only let to do is to update these siggys to match the ones on the README.
(no `e` required, that is for `Maybe`)